SU-SWS/stanford_profile_helper
# 8.1.10

# READY FOR REVIEW

# Summary
- 8.1.10 hotfix to lock basic pages to 3 per row without affecting existing pages.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. checkout the `master` branch and create a basic page with 4 items per row
2. checkout this branch and clear caches
3. edit that page above and verify you still have the 4 per row, but can only do 3 per row on other rows now.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
